Ixfleura/Program.cs

```diff
@@ -67,6 +93,7 @@ static void Main(string[] args)
 
             try
             {
+                host.Services.GetService<CampaignService>();    // Initialise campaign service
```

Comment:
This might be considered really bad practice, but the CampaignService needs to have its constructor run at startup somehow.